packages/test/stochastic-test-utils/package.json

```diff
@@ -20,22 +20,6 @@
 		"./test/utils": {
 			"types": "./dist/test/utils.d.ts",
 			"default": "./dist/test/utils.js"
-		},
-		"./alpha": {
-			"types": "./dist/stochastic-test-utils-alpha.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./beta": {
-			"types": "./dist/stochastic-test-utils-beta.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./internal": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./public": {
-			"types": "./dist/stochastic-test-utils-public.d.ts",
-			"default": "./dist/index.js"
```

Comment:
This package is scoped '@fluid-private', which means that eslint will complain when importing from '/internal' (my exemption was for @fluid-framework/*/internal).

We import this package *a lot*, and it's already advertised as private, so leaving untrimmed for now.